Update decentlab devices

decentlab opened the pull request:
<!--
Thanks for submitting a pull request. Please fill the template below,
otherwise we will not be able to process this pull request.
-->

#### Summary
<!--
A short summary, referencing related issues:
Closes #0000, References #0000, etc.
-->

Adds devices and photos

#### Changes
<!-- What are the changes made in this pull request? -->

- added dl-ntu
- added dl-lws
- added dl-itst-002
- added dl-wrm-002
- added additional photos for other devices
- tweaked vendor info links

#### Notes for Reviewers
<!--
NOTE: This section is optional.

Motivate briefly why it is implemented this way, if that deviates from the
implementation proposal in the referenced issues.
- How should your reviewers approach this pull request?
- @mention reviewers with special requests or questions for them
-->

- added two new sensors `leaf wetness` and `turbidity`

#### Release Notes
<!--
NOTE: This section is optional.

Any notes that we need to include in the Release Notes for the next release.
These notes are formatted as bullet points, written in past tense, and will be
combined with the labels of this Pull Request.

Always mention changes in API.
-->

Jaime-Trinidad commented on lib/draft/schema.json:
```diff
@@ -958,15 +960,7 @@
-              "analog",
-              "dali"
-            ]
+            "enum": ["modbus", "m-bus", "can bus", "rs-485", "sdi-12", "analog"]
```
please leave them in a list form for order, and `dali` is missing

decentlab replied:
Fixed the mistake.

NB: also added downlink codecs.

Jaime-Trinidad commented:
@decentlab thanks for the updates, there is one comment and please make a second check in the images, the ones that have background aren't the `main` images, main images should be without background for the device page.

decentlab commented:
@Jaime-Trinidad, thanks for the review.

- fixed the lib/draft merge mistake
- checked all main photos for PNG extension.
- added downlink codecs.
